gpt_index/indices/tree/base.py

```diff
@@ -67,6 +68,7 @@ def __init__(
         self.summary_template = summary_template or DEFAULT_SUMMARY_PROMPT
         self.insert_prompt: TreeInsertPrompt = insert_prompt or DEFAULT_INSERT_PROMPT
         self.build_tree = build_tree
+        self._async_ = async_
```

Comment:
since `async_` is user-facing, could we name this to something more friendly? e.g. `use_async` or something

Reply:
agreed